nutils/SI.py

```diff
@@ -15,7 +15,7 @@
     >>> from nutils import SI
     >>> v = SI.parse('7μN*5h/6g')
 
-The Quantity constructor recognizes the multiplication (\*) and division (/)
+The Quantity constructor recognizes the multiplication (\\*) and division (/)
```

Comment:
This is to prevent interpreting the asterisk as the start of a span of italic text. See https://docutils.sourceforge.io/docs/user/rst/quickstart.html#toc-entry-2 .